src/BankingApp.Data/AccountDbContext.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Principal;
+using System.Text;
+using System.Threading.Tasks;
+using BankingApp.Data.Entities;
+using Microsoft.EntityFrameworkCore;
+
+namespace BankingApp.Data
+{
+
+    public class AccountDbContext : DbContext
+    {
+     
```

Comment:
Please, remove this spaces. Since you are just creating the class for now. I also observed you didn't create the entity class "Account"

Reply:
The entity classes will come in a different task. Good call out.